styles/PageTitle.module.css

```diff
@@ -0,0 +1,48 @@
+.container {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    flex-wrap: nowrap;
+    margin-bottom: 50px;
+    cursor: pointer;
+    width: 100%;
+}
+
+.titleBox {
+    width: 25%;
+    padding: 0 20px 0 20px;
+    background-image: linear-gradient(to bottom, 
+    white 40%, #94C7E7 40% 70%, white 70%);
+}
+
+.titleBox h1 {
+    margin-top: 0;
+    font-weight: 500;
+    font-size: 50px;
+    text-align: center;
+}
+
+.moon {
+    height: 40px;
+    width: 40px;
+    align-self: flex-start;
+}
+
+@media (max-width: 1140px) {
+
+    
+    .container {
+        padding: 0 0 0 0;
```

Comment:
Minor, but just so you know you can just do `padding: 0;` , will override each side if its just one number